signer/plugin.go

```diff
@@ -87,6 +87,7 @@ func (s *pluginSigner) generateSignature(ctx context.Context, desc ocispec.Descr
 		},
 	}
 
+	opts.SigningAgent = fmt.Sprintf("%s;%s/%s", signingAgent, metadata.Name, metadata.Version)
```

Comment:
You may consider using the same format as [User Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent)

Reply:
Not sure that format directly applies to the info we're wanting to convey here. Unless you feel strongly about it, would prefer to keep as-is

Reply:
@yizha1 @FeynmanZhou @priteshbandi for comments

Reply:
Yeah, lets create an issue (https://github.com/notaryproject/notation-go/issues/225) to discuss about using browser like user agent for rc2. Since PR is improvement over the existing hard coded user agent, can we please go ahead and merge it?

Reply:
@jondonas updated the format to abide by [rfc9110.html#field.user-agent](https://httpwg.org/specs/rfc9110.html#field.user-agent)